lua-guidelines.md

```diff
@@ -66,19 +66,18 @@ need_*, should_*, must_*, in_*, not_* и т.п.
 для функций, не имеющих отношения к созданию объектов. Для таких функций
 рекомендуется префикс create_.
 
+##Кавычки
+
+Рекомендуемые кавычки по-умолчанию - апострофы. Если в строке содержатся
+апострофы - использовать для данной строки кавычки по своему усмотрению.
+
 ##Пробельные символы
 
 В коде **нельзя использовать символы табуляции** (кроме мэйкфайлов). Если в
 строковом литерале нужен символ табуляции, используйте
 
     "\t".
 
-Строки желательно ограничивать теми кавычками, которых нет в тексте:
-
-    "\"" -> '"'
-    '\'' -> "'"
-    "\"'" -> [["']]
-
```

Comment:
I'm not inspired by guidelines changes that 1) expect all codebase to be rewritten 2) without a good cause. But as recommendation - ok.
Also why example was removed?